### PR TITLE
Adding maintainer docs

### PR DESCRIPTION
**Pull request by Ellerbach**
## Description

Adding maintainer docs

## Motivation and Context

- Adding maintainer docs

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [x] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [x] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: maintainer.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    - XIAO_ESP32C3
+    - ESP32_C3_REV3
+```
+
+> [!Important]
+> List all the firmware that are present in existing self hosted agents.
+
+This will be adjusted with a git action later to browse existing agents and gather this information. So far, it's a manual gathering.
+
+Transform the pool into a multi stage, don't forget to indend everything:
```
Comment (josesimoes) [inline maintainer.md:71]: maybe "include" instead?
```suggestion
Transform the pool into a multi stage, don't forget to include everything:
```

**Review by josesimoes — APPROVED** [review]
LGTM! just a small typo.

BTW: should be the check warnings that are shoing here from code analysis be addressed on a new PR?

File: maintainer.md (hunk 1 above)
Comment (josesimoes) [reply maintainer.md:71]: or maybe indent.... 😅

Comment (Ellerbach) [reply maintainer.md:71]: it's definitely indent!

**Comment by Ellerbach** [comment]
> BTW: should be the check warnings that are shoing here from code analysis be addressed on a new PR?

Yes, 100%, just want to get something stable and working before fixing the warnings (most are very straight forward)

**Pull request merged**